helpers/imageUpload.js

```diff
@@ -7,7 +7,7 @@ function imageUpload(){
         fileUpload
     );
     fileUpload.waitForDisplayed();
-    const filePath = path.join(__dirname, "../data/123.jpeg");
+    const filePath = path.join(__dirname, "../data/Bug.jpg");
```

Comment:
The function should not use the hardcoded path

Reply:
need more clarification please: if i have to change the path here, how do i do that?